docs/reference/edot-python/configuration.md

```diff
@@ -108,7 +108,7 @@ EDOT Python uses different defaults than OpenTelemetry Python for the following
 | `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` | `DELTA` | `CUMULATIVE` | |
 | `OTEL_LOG_LEVEL` | `warn` | | {applies_to}`edot_python: ga 1.9.0` |
 | `OTEL_METRICS_EXEMPLAR_FILTER` | `always_off` | `trace_based` | |
-| `OTEL_TRACES_SAMPLER` | `parentbased_traceidratio` | `parentbased_always_on` | {applies_to}`edot_python: ga 1.5.0` |
+| `OTEL_TRACES_SAMPLER` | `experimental_composite_parentbased_traceidratio` | `parentbased_always_on` | {applies_to}`edot_python: ga 1.10.0` |
```

Comment:
@theletterf Since 1.10.0 is not out yet, what's a good way to say "`parentbased_traceidratio` since 1.5.0, `experimental_composite_parentbased_traceidratio` since 1.10.0"?

Reply:
How about

```suggestion
| `OTEL_TRACES_SAMPLER` | `experimental_composite_parentbased_traceidratio` | `parentbased_always_on` | {applies_to}`edot_python: ga 1.10.0` |
| `OTEL_TRACES_SAMPLER` | `parentbased_traceidratio` | `parentbased_always_on` | {applies_to}`edot_python: ga 1.5.0, deprecated 1.10.0` |
```